isic/core/search.py

```diff
@@ -129,6 +129,8 @@ def bulk_add_to_search_index(qs: QuerySet) -> None:
         client=get_elasticsearch_client(),
         index=settings.ISIC_ELASTICSEARCH_INDEX,
```

Comment:
I can't find any docs on this `index` parameter to `streaming_bulk`. Do you have a good reference?

Reply:
Internally it passes it to `client.bulk` - https://elasticsearch-py.readthedocs.io/en/7.x/api.html#elasticsearch.Elasticsearch.bulk.